config/routes.rb

```diff
@@ -94,10 +94,11 @@
   get '/landing' => 'landing#index'
 
   # map
-  get '/map' => 'map#index'
-  get '/map/*path' => 'map#index'
-  get '/embed/map' => 'map#embed'
-  get '/embed/map/*path' => 'map#embed'
+  get '/map' => 'map_legacy#index'
+  get '/map/*path' => 'map_legacy#index'
+  get '/embed/map' => 'map_legacy#embed'
+  get '/embed/map/*path' => 'map_legacy#embed'
+  get '/v2/map' => 'map#index'
```

Comment:
Can we call this /map/v2 ? It just doesnt seem right this way around 😁

Reply:
fine with me